app/helpers/react_helper.rb

```diff
@@ -37,6 +37,7 @@ def page_data(affiliate)
     {
       affiliate: affiliate.name,
       display_logo_only: affiliate.display_logo_only,
+      showVoteOrgLink: affiliate.show_vote_org_link && Rails.application.secrets[:enable_vote_org_link],
```

Comment:
We're moving away from secrets. As it says in the ticket, we need to use the `ENABLE_VOTE_ORG_LINK` environment variable instead.